src/ui/components/address-action/TransactionSimulation/TransactionSimulation.tsx

```diff
@@ -106,7 +106,7 @@ export function TransactionSimulation({
   const txInterpretQuery = useQuery({
     enabled: FEATURE_PAYMASTER_ENABLED
       ? shouldInterpretTransaction({ network, eligibilityQuery })
```

Comment:
Should we add check for `supports_simulations` when `FEATURE_PAYMASTER_ENABLED=on`?